sn_transfers/src/wallet/local_store.rs

```diff
@@ -62,7 +61,7 @@ impl LocalWallet {
 
     /// Stores the given cash_notes to the `created cash_notes dir` in the wallet dir.
     /// These can then be sent to the recipients out of band, over any channel preferred.
-    pub fn store_cash_notes(&self, cash_note: Vec<&CashNote>) -> Result<()> {
+    pub fn store_cash_notes_to_disk(&self, cash_note: Vec<&CashNote>) -> Result<()> {
```

Comment:
Can we rename the above `store_cash_note` to `store_cash_note_to_disk` or maybe remove it?